haas/client/user.py

```diff
@@ -50,3 +50,22 @@ def remove(self, user, project):
         return self.check_response(
                 self.httpClient.request("POST", url, data=payload)
                 )
+
+    def set_admin(self, username, privilege):
```

Comment:
I do not like that we're introducing magic strings here. This should accept a True or False for is_admin and apply that.